kernel/nativevalues.ml

```diff
@@ -90,8 +92,7 @@ type atom =
   | Acase of annot_sw * accumulator * t * t
   | Afix of t array * t array * rec_pos * int
             (* types, bodies, rec_pos, pos *)
-  | Acofix of t array * t array * int * t
-  | Acofixe of t array * t array * int * t
+  | Acofix of t array * t array * int * vcofix
```

Comment:
Would it make sense to use an inline record here?
```ocaml
Acofix of { ...; mutable vcofix: vcofix }
```
That way, we would no longer need to use `set_atom_of_accu`, we would just do `co.vcofix <- CofixValue v` when updating. Or is it important that a fresh `Acofix` allocation is performed?

Reply:
AFAIK it doesn't matter that we allocate a new atom. We could indeed share the computation further, but I'm a bit reluctant to expose the mutable interface to the outside. Rather, I'd prefer to wrap internally the vcofix type into a reference.